CloudCulator/Cloudculator.Core.UnitTests/CalculatorEngineUnitTests.cs

```diff
@@ -0,0 +1,134 @@
+﻿using Cloudculator.Core.Engines;
+
+namespace Cloudculator.Core.UnitTests
+{
+    public class CalculatorEngineUnitTests
+    {
+        private CalculatorEngine service;
+
+        public CalculatorEngineUnitTests()
+        {
+            service = new CalculatorEngine();
```

Comment:
Since it is private field, please use underscore notation: _service

Reply:
Done